geowebcache/azureblob/src/main/java/org/geowebcache/azure/AzureClient.java

```diff
@@ -93,8 +93,13 @@ public AzureClient(AzureBlobStoreData configuration) throws StorageException {
             this.container = serviceURL.createContainerURL(containerName);
             // no way to see if the containerURL already exists, try to create and see if
             // we get a 409 CONFLICT
+            int status;
```

Comment:
Tested interactively, I can't get the code to throw an exception? Different azure container setup maybe? Or a difference in dependencies?

Reply:
My first thought was it would be a difference in dependencies, tried everything I could think of, but the only thing that worked was reverting gwc 1.26-SNAPSHOT to 1.25-SNAPSHOT.
So I debugged both and saw the difference, the patch is not in the 1.25.x branch.
BTW the exception happens against the Azurite test container when running integration tests in gscloud

Reply:
Ok so you confirm the issue is not happening against an actual Azure storage, but the Azurite test container. Would be nice if the title was clear about it.

Reply:
Hi, I actually confirm the issue happens both with Azure and Azurite. The only difference is you may be always running the tests against an Azure container that already exists, whereas the Azurite test has to create it.
If you try a container name that does not exist in Azure you should get the same error.